plugins/fclrave/fclcollision.h

```diff
@@ -1225,7 +1237,7 @@ class FCLCollisionChecker : public OpenRAVE::CollisionCheckerBase
             LinkConstPtr plink = link_raw->GetLink();
             if( !plink ) {
                 if( link_raw->bFromKinBodyLink ) {
-                    RAVELOG_WARN_FORMAT("The link %s was lost from fclspace (env %d) (userdatakey %s)", GetEnv()->GetId()%link_raw->bodylinkname%_userdatakey);
+                    RAVELOG_WARN_FORMAT("The link %s was lost from fclspace (env %d) (userdatakey %s)", link_raw->bodylinkname%GetEnv()->GetId()%_userdatakey);
```

Comment:
I am fixing the former suggestion using `GetEnv()->GetNameId()`.
For the latter part, I don't think we can print the parent body name since `plink` is `nullptr` and `link_raw` does not have parent link information so far.

Reply:
We can get `link_raw->bodylinkname` because `bodylinkname` in `FCLSpace::KinBodyInfo::LinkInfo` stores it for debugging purpose. Maybe we can store `parentbodyname` when constructed?

Reply:
Parent body was already included.